### ISSUE TITLE
Bump 1.0

### ISSUE THREAD

**Issue by kellyselden** [embroider-build/embroider#761]
Now that this shows up in the default blueprint (currently beta https://github.com/ember-cli/ember-new-output/compare/v3.26.0...v3.27.0-beta.1), can we bump it to 1.0? I think it would increase confidence if it had semver guarantees.

**Comment by rwjblue** [comment]
Yes! We are working on the final list of things that we thing would be required before 1.0.0, hopefully we'll have a specific meta issue for that up this week.

It may not be 1.0.0 before we ship ember-cli@3.27.0 final (depends on that list, and folks' time to handle the things we identify), but hopefully it will be before 3.28.0 (an LTS release).

**Comment by bertdeblock** [comment]
This one can be closed I think.
